<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Timeline" name="timeline" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionSearchFiles"/>
    <addaction name="actionComputeHashes"/>
   </widget>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionFaster"/>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionSlower"/>
   <addaction name="actionNext"/>
   <addaction name="actionSetStart"/>
   <addaction name="actionSetStop"/>
   <addaction name="actionSearchFiles"/>
   <addaction name="actionRandomPlay"/>
   <addaction name="actionShowNextOSD"/>
   <addaction name="actionShowInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Playlist</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableView" name="playTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/document.svg</normaloff>:/image/document.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-play.svg</normaloff>:/image/media-play.svg</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-stop.svg</normaloff>:/image/media-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-skip-backward.svg</normaloff>:/image/media-skip-backward.svg</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-skip-forward.svg</normaloff>:/image/media-skip-forward.svg</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSetStart">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/collapse-left.svg</normaloff>:/image/collapse-left.svg</iconset>
   </property>
   <property name="text">
    <string>Set Start</string>
   </property>
  </action>
  <action name="actionSetStop">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/collapse-right.svg</normaloff>:/image/collapse-right.svg</iconset>
   </property>
   <property name="text">
    <string>Set Stop</string>
   </property>
  </action>
  <action name="actionSearchFiles">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/project.svg</normaloff>:/image/project.svg</iconset>
   </property>
   <property name="text">
    <string>Search Files</string>
   </property>
  </action>
  <action name="actionRandomPlay">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/random.svg</normaloff>:/image/random.svg</iconset>
   </property>
   <property name="text">
    <string>Random Play</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionShowNextOSD">
   <property name="text">
    <string>Show Next OSD</string>
   </property>
  </action>
  <action name="actionShowInfo">
   <property name="text">
    <string>Show Info</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionComputeHashes">
   <property name="text">
    <string>Compute Hashes</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-step-forward.svg</normaloff>:/image/media-step-forward.svg</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionFaster">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/arrow-circle-top.svg</normaloff>:/image/arrow-circle-top.svg</iconset>
   </property>
   <property name="text">
    <string>Faster</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionSlower">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/arrow-circle-bottom.svg</normaloff>:/image/arrow-circle-bottom.svg</iconset>
   </property>
   <property name="text">
    <string>Slower</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="../qxxx.qrc">
     <normaloff>:/image/media-step-backward.svg</normaloff>:/image/media-step-backward.svg</iconset>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Timeline</class>
   <extends>QWidget</extends>
   <header>src/timeline.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qxxx.qrc"/>
 </resources>
 <connections/>
</ui>
